src/views/Notifications/index.js

```diff
@@ -1,118 +1 @@
-
-import styles from './styles.scss';
-
-const propTypes = {
-    // eslint-disable-next-line react/forbid-prop-types
-    notifications: PropTypes.array.isRequired,
-
-    activePage: PropTypes.number.isRequired,
-    notificationsPerPage: PropTypes.number.isRequired,
-
-    setNotifications: PropTypes.func.isRequired,
-};
-
-const defaultProps = {
-};
-
-const mapStateToProps = state => ({
-    notifications: notificationsSelector(state),
-    activePage: notificationsActivePageSelector(state),
-    notificationsPerPage: notificationsPerPageSelector(state),
-});
-
-const mapDispatchToProps = dispatch => ({
-    setNotifications: params => dispatch(setNotificationsAction(params)),
-});
-
-@connect(mapStateToProps, mapDispatchToProps)
-export default class Notifications extends React.PureComponent {
-    static propTypes = propTypes;
-    static defaultProps = defaultProps;
-    static notificationKeyExtractor = n => n.key;
-    static notificationItems = {
-        project_join_request: ProjectJoin,
-    }
-
-    constructor(props) {
-        super(props);
-        this.state = {
-            notificationsLoading: true,
-        };
-
-        this.notificationsRequest = new NotificationsGetRequest({
-            setState: d => this.setState(d),
-            setNotifications: props.setNotifications,
-        });
-    }
-
-    componentDidMount() {
-        const {
-            activePage,
-            notificationsPerPage,
-        } = this.props;
-
-        this.notificationsRequest.init({
-            activePage,
-            notificationsPerPage,
-        });
-        this.notificationsRequest.start();
-    }
-
-    renderNotificationItem = (key, data) => {
-        const NotificationItem = Notifications.notificationItems[data.type];
-
-        return (
-            <NotificationItem
-                key={key}
-                data={data}
-            />
-        );
-    }
-
-    renderEmptyComponent = () => (
-        <div className={styles.emptyComponent} >
-            {_ts('notifications', 'noNotificationsText')}
-        </div>
-    );
-
-    render() {
-        const { notificationsLoading } = this.state;
-        const { notifications } = this.props;
-
-        return (
-            <div className={styles.notifications} >
-                {notificationsLoading && <LoadingAnimation />}
-                <header className={styles.header} >
-                    <h2 className={styles.heading} >
-                        {_ts('notifications', 'notificationHeaderTitle')}
-                    </h2>
-                </header>
-                <ListView
-                    className={styles.content}
-                    data={notifications}
-                    keySelector={Notifications.notificationKeyExtractor}
-                    modifier={this.renderNotificationItem}
-                    emptyComponent={this.renderEmptyComponent}
-                />
-            </div>
-        );
-    }
-}
+// TODO: delete this file
```

Comment:
Y dough?

Reply:
Safar's request to not change `routes` file which has reference to this view, but I had already moved this `view` to the `components`